### PR TITLE
feat: add quiz for chapter09-01

### PR DESCRIPTION
**Pull request by Ejembiii**
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/793)

### CODE REVIEW

File: quizzes/ch09-01-unrecoverable-errors-with-panic.toml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+[[questions]]
+id = "695f318f-539b-4ce8-9c00-92e29f7f1bb3"
+type = "MultipleChoice"
+prompt.prompt = """
+In Cairo, the panic function can be triggered:
```
Comment (enitrat) [inline quizzes/ch09-01-unrecoverable-errors-with-panic.toml:5]: check the syntax of the question, there are issues with your phrasing

File: quizzes/ch09-01-unrecoverable-errors-with-panic.toml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+3. A syntactic scope.
+4. Deliberately, by invoking the panic function.
+"""
+answer.answer = "1 and 4"
+prompt.distractors = [ "1 only", "2 only", "2 and 3" ]
+context = """
+There are two primary ways that a panic can be triggered in Cairo: inadvertently, through actions causing the code to panic (e.g., accessing an array beyond its bounds), or deliberately, by invoking the panic function.
+"""
+
+[[questions]]
```
Comment (enitrat) [inline quizzes/ch09-01-unrecoverable-errors-with-panic.toml:17]: Delete the two last questions, they're useless

**Review by enitrat — CHANGES_REQUESTED** [review]

File: quizzes/ch09-01-unrecoverable-errors-with-panic.toml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+When a panic occurs, it leads to an abrupt termination of the program.
+"""
+
+[[questions]]
+id = "695f318f-539b-4ce8-9c00-92e29f7f1bb3"
+type = "MultipleChoice"
+prompt.prompt = """
+In Cairo, the ```panic``` function can be triggered in two ways:
+1. Inadvertently, through actions causing the code to panic.
+2. A statement.
```
Comment (enitrat) [inline quizzes/ch09-01-unrecoverable-errors-with-panic.toml:18]: "A statement" is not a well formed sentence in the context of the question. Pay attention to the quality!

File: quizzes/ch09-01-unrecoverable-errors-with-panic.toml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+prompt.distractors = [ "1 only", "2 only", "2 and 3" ]
+context = """
+There are two primary ways that a panic can be triggered in Cairo: inadvertently, through actions causing the code to panic (e.g., accessing an array beyond its bounds), or deliberately, by invoking the panic function.
+"""
+
+[[questions]]
+id = "695f318f-539b-4ce8-9c00-92e29f7f1bb3"
+type = "MultipleChoice"
+prompt.prompt = """
+What does the ```panic_with``` attribute used for in Cairo?"""
```
Comment (enitrat) [inline quizzes/ch09-01-unrecoverable-errors-with-panic.toml:32]: the sentence is not written in good english, neither are the answers

**Review by enitrat — CHANGES_REQUESTED** [review]

**Comment by Ejembiii** [comment]
@enitrat I've made the changes you requested

**Review by enitrat — DISMISSED** [review]

**Review by enitrat — APPROVED** [review]

**Pull request merged**